ee/api/test/test_organization_resource_access.py

```diff
@@ -0,0 +1,119 @@
+from rest_framework import status
+
+from ee.api.test.base import APILicensedTest
+from ee.models.organization_resource_access import OrganizationResourceAccess
+from posthog.models.organization import OrganizationMembership
+
+
+class TestOrganizationResourceAccessAPI(APILicensedTest):
+    def setUp(self):
+        super().setUp()
+
+    def test_only_organization_admins_and_higher_can_set_resource_access(self):
+        self.organization_membership.level = OrganizationMembership.Level.ADMIN
+        self.organization_membership.save()
+        admin_create_res = self.client.post(
+            "/api/organizations/@current/resource_access",
+            {
+                "resource": OrganizationResourceAccess.Resources.FEATURE_FLAGS,
+            },
+        )
+        self.assertEqual(admin_create_res.status_code, status.HTTP_201_CREATED)
+        get_res = self.client.get("/api/organizations/@current/resource_access")
+        self.assertEqual(get_res.json()["count"], 1)
+        self.assertEqual(get_res.json()["results"][0]["resource"], OrganizationResourceAccess.Resources.FEATURE_FLAGS)
+
+        self.organization_membership.level = OrganizationMembership.Level.MEMBER
+        self.organization_membership.save()
+        member_create_res = self.client.post(
+            "/api/organizations/@current/resource_access",
+            {
+                "resource": OrganizationResourceAccess.Resources.EXPERIMENTS,
+            },
+        )
+        self.assertEqual(member_create_res.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_can_only_create_one_instance_of_each_resource_type(self):
```

Comment:
would be nice to have multiple orgs in this test

Reply:
✅